cluster/clean.sh

```diff
@@ -36,6 +36,7 @@ set +e
     _kubectl -n ${namespace} delete kv kubevirt
 )
 _kubectl -n ${namespace} patch kv kubevirt --type=json -p '[{ "op": "remove", "path": "/metadata/finalizers" }]'
+_kubectl patch cdi cdi --type=json -p '[{ "op": "remove", "path": "/metadata/finalizers" }]'
```

Comment:
You must be getting into this condition because the cdi operator is deleted before the CDI CR.  So, why not explicitly delete the CR here instead of the finalizer?  And let the operator do its thing?  Ideally, the loops that follow this line should be unnecessary for CDI stuff.  I suspect we are still leaving some stuff around right now (bugs) and it doesn't hurt to have them but maybe it shouldn't be the main way of cleaning up CDI?

Reply:
This clean script is intentionally doing a "brute force" way of deleting things, and not a graceful shutdown. Useful during development, when you just want to be sure that everything is deleted, no matter if things works as expected or not.